src/sentry/static/sentry/app/components/customIgnoreDurationModal.tsx

```diff
@@ -104,24 +109,22 @@ export default class CustomIgnoreDurationModal extends React.Component<Props, St
               />
             </div>
           </form>
-        </div>
+        </Modal.Body>
         {this.state.dateWarning && (
-          <div className="alert alert-error" style={{marginTop: '5px'}}>
+          <Alert icon={<IconWarning size="md" />} type="error">
             {t('Please enter a valid date in the future')}
-          </div>
+          </Alert>
         )}
-        <div className="modal-footer">
-          <button
-            type="button"
-            className="btn btn-default"
-            onClick={this.props.onCanceled}
-          >
-            {t('Cancel')}
-          </button>
-          <button type="button" className="btn btn-primary" onClick={this.snoozeClicked}>
-            {t('Ignore')}
-          </button>
-        </div>
+        <Modal.Footer>
+          <ButtonBar gap={1}>
+            <Button priority="default" onClick={this.props.onCanceled}>
+              {t('Cancel')}
+            </Button>
```

Comment:
Does this need `type=button`? It might have unexpected behavior with the form otherwise (it should probably be the default type for `<Button>`'